utilities/message_filters/include/message_filters/time_sequencer.h

```diff
@@ -213,22 +213,22 @@ class TimeSequencer : public SimpleFilter<M>
     }
   }
 
-  void update(const ros::TimerEvent&)
+  void update(const ros::SteadyTimerEvent&)
   {
     dispatch();
   }
 
   void init()
   {
-    update_timer_ = nh_.createTimer(update_rate_, &TimeSequencer::update, this);
+    update_timer_ = nh_.createSteadyTimer(ros::WallDuration(update_rate_.toSec()), &TimeSequencer::update, this);
```

Comment:
Can `WallDuration` really not be initialized from a `Duration`? That seems like an API fail.

Reply:
IIRC it can't... but I didn't check that again and did it explicitly just to be sure...